extensions/mssql/src/utils.ts

```diff
@@ -223,7 +223,7 @@ export function getErrorMessage(error: Error | any, removeHeader: boolean = fals
 	if (error instanceof Error) {
 		errorMessage = error.message;
 	} else if (error.responseText) {
-		errorMessage = error.responseText;
+		errorMessage = error.responseText.toString() as string;
```

Comment:
why adding the `.toString()` here?

Reply:
Because error.responseText could be anything. It should always be a string (and was previously assumed to be so) so technically could get rid of it I guess. (yet again I really wish TS had better built in error support...)